gammapy/modeling/models/tests/test_spatial.py

```diff
@@ -720,11 +721,14 @@ def test_piecewise_spatial_model():
         geom = WcsGeom.create(
             skydir=(lon, 2.3), npix=(2, 2), binsz=0.3, frame="galactic"
         )
+
         coords = MapCoord.create(geom.footprint)
+        print(coords["lon"])
```

Comment:
could you remove the print statements?

Reply:
done in https://github.com/gammapy/gammapy/pull/5309